src/views/activity/activity.view.tsx

```diff
@@ -28,13 +28,11 @@ const Activity: FC = () => {
 
   const onClaim = async (tx: Transaction) => {
     if (tx.status === "on-hold") {
-      if (!(await isConnectedProviderChainOk(tx.bridge.destinationNetwork))) {
+      if (isConnectedProviderChainOk && !isConnectedProviderChainOk(tx.bridge.destinationNetwork)) {
```

Comment:
I think that `isConnectedProviderChainOk` can't be `null` or `undefined` right?

Reply:
Yes, that one keeps as before with await

Reply:
But why do you check if `isConnectedProviderChainOk` exists then? And why are not we awaiting for the result as before if the function still returns a `Promise`?